package.json

```
@@ -36,9 +36,12 @@
     "typescript": "^4.7.3"
   },
   "scripts": {
+    "build:all:lowmem": "NODE_OPTIONS='--max-old-space-size=4096' npm run build:all",
     "build:all": "yarn run concurrently --success=all -r -m=1 'yarn workspaces-to-typescript-project-references' 'yarn turbo run prepublish'",
     "test": "yarn run concurrently --success=all -r -m=1 'yarn workspaces-to-typescript-project-references --check' 'yarn turbo run test'",
     "test:watch": "yarn turbo run test:watch",
+    "test:update": "yarn turbo run test:update",
+    "reformat:all": "yarn turbo run reformat",
```

```suggestion
    "format": "yarn prettier --write '**/*.{ts,md}'",
```
I think just one command like this is fine. It's not necessary to add one for all sub-packages.